src/python/WMCore/WMExceptions.py

```diff
@@ -22,37 +22,274 @@
+        60306: "Failed to copy and register output file.",
+        60307: "Failed to copy an output file to the SE (sometimes caused by timeout issue).",
+        60308: "An output file was saved to fall back local SE after failing to copy to remote SE.",
+        60309: "Failed to create an output directory in the catalogue.",
+        60310: "Failed to register an output file in the catalogue.",
+        60311: "Local Stage Out Failure using site specific plugin.",
+        60312: "Failed to get file TURL via lcg-lr command.", # Mismatch...  WMCore raises if it is unable to load StageIn/Out/Delete impl. and in another place also if No logs staged in during LogCollect step.
+        60313: "Failed to delete the output from the previous run via lcg-del command.", # Mh.. only lcg-del?
+        60314: "Failed to invoke ProdAgent StageOut Script.",
+        60315: "ProdAgent StageOut initialisation error (Due to TFC, SITECONF etc).",
+        60316: "Failed to create a directory on the SE.",
+        60317: "Forced timeout for stuck stage out.",
+        60318: "Internal error in Crab cmscp.py stageout script.", # Mismatch... WMCore raise this if it get HTTPError or Execption uploading data to DQM
+        60319: "Failure to do asynchronous stageout.", # Mismatch... WMCore raises if there is AlcaHarvestFailure during copy to EOS
+        60320: "Failure to communicate with ASO server.",
+        60321: "Site related issue: no space, SE down, refused connection.",
+        60322: "User is not authorized to write to destination site.",
+        60323: "User quota exceeded.",
+        60324: "Other stageout exception.",
+        60401: "Failure to assemble LFN in direct-to-merge by size (WMAgent).",
+        60402: "Failure to assemble LFN in direct-to-merge by event (WMAgent).",
+        60403: "Timeout during attempted file transfer- status unknown (WMAgent).",
+        60404: "Timeout during staging of log archives- status unknown (WMAgent).",
+        60405: "General failure to stage out log archives (WMAgent).",
+        60406: "Failure in staging in log files during log collection (WMAgent).",
+        60407: "Timeout in staging in log files during log collection (WMAgent).",
+        60408: "Failure to stage out of log files during log collection (WMAgent).",
+        60409: "Timeout in stage out of log files during log collection (WMAgent).",
+        60410: "Failure in deleting log files in log collection (WMAgent).",
+        60411: "Timeout in deleting log files in log collection (WMAgent).",
+        60450: "No output files present in the report",
+        60451: "Output file lacked adler32 checksum (WMAgent).",
+        60452: "No run/lumi information in file (WMAgent).", # Wrong exitCode
+        60453: "Unable to determine pset hash from output file (CRAB3).", # Wrong exitCode
+        60514: "PreScriptFailure.", # Wrong exitCode
+        60515: "PreScriptScramFailure.", # Wrong exitCode
```

Comment:
I have a patch to remove them and replace everything scram related (that I found) with 50513. I think a lot of this 60xxx stuff snuck in as a typo.